connector/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/config/UserRegistryConnectorConfig.java

```diff
@@ -0,0 +1,9 @@
+package it.pagopa.selfcare.dashboard.connector.rest.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource(value = "classpath:config/user-registry-connector.properties")
```

Comment:
@KevinSi96  why should you load an empty properties file (`connector/rest/src/main/resources/config/user-registry-connector.properties`)? If it isn't required, also remember to remove that file